external-docs/docs/guides/configuration/qube-config.md

```diff
@@ -115,66 +120,560 @@ To reuse or extend existing dimensions, attributes, units, or measures, provide
 | `describes_observations` | Associates this attribute with the relevant observation values. This is only necessary for [pivoted shape data sets](../shape-data/pivoted-shape.md) with multiple observation value columns.                                                                                                                                                                                  | *none*                                                                           |
 | `required`               | If this boolean value is true csvcubed will flag to the user if there are blank values in this column                                                                                                                                                                                                                                                                          | *none*                                                                           |
 | `data_type`              | (Attribute Literals only) The [xml data type](https://www.w3.org/TR/xmlschema-2/#built-in-datatypes) of the contents of the column, if this is provided it becomes a Literal Attribute column (Optional)                                                                                                                                                                       | *none*                                                                           |
 | `values`                 | (New Resource Attributes only) If automatically-generated attributes are desired, a boolean value of `true` is used to signify to csvcubed to create attribute resources from values in this column; otherwise this should be a dictionary defining the attributes used in the column. See [Attribute values configuration](#attribute-values-configuration) for more details. | *none*                                                                           |
 | `cell_uri_template`      | (Existing Resource Attributes only) Used to define a template to map the cell values in this column to URIs                                                                                                                                                                                                                                                                    | *none*                                                                           |
 
+### Examples of attributes configuration
+
+
+For the purposes of these instructions, we will be using the `Number of Arthur's Bakes` data set.
+
+#### Standard shape
```

Comment:
I don't think we need to distinguish between data shapes in this documentation. Can you remove references to it to simplify?

I'd be tempted to just run with the multi-measure pivoted without the standard shape tbh. If they accidentally use `describes_observations` in the standard shape it won't cause any harm.

Reply:
I've tidied up the examples in attributes.md